cw/tvla.py

```diff
@@ -827,6 +814,19 @@ def run_tvla(ctx: typer.Context):
             single_trace = np.load(single_trace_file)
             assert num_samples == single_trace.shape[0]
 
+    if save_to_disk_ttest:
+        if num_steps > 1:
```

Comment:
Good point. I've changed it now so that `save_to_disk_ttest` is generated as:
`save_to_disk_ttest = cfg["save_to_disk_ttest"] and (cfg["ttest_step_file"] is None)`